src/main/java/com/google/alpollo/SongDataBase.java

```diff
@@ -2,20 +2,23 @@
 
 import java.util.List;
 import com.google.alpollo.model.AnalysisInfo;
```

Comment:
Changes to this file were already merged in PR #51.
Can you please rebase this change to make it easier to review?

Reply:
nevermind. I realized the other PRs were merged into the branch of this PR. I'll review it now.